parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -55,6 +55,7 @@ ERR_CIS_APPLY_PASSWORD_CONFIG=115 {{/* Error applying CIS-recommended passwd con
 
 ERR_VHD_FILE_NOT_FOUND=124 {{/* VHD log file not found on VM built from VHD distro */}}
 ERR_VHD_BUILD_ERROR=125 {{/* Reserved for VHD CI exit conditions */}}
+ERR_APMZ_INSTALL_ERROR=126 {{/* Unable to install apmz telemetry client */}}
```

Comment:
Are you planning on having this be a fatal error?
I've been debating what we should do if we can't get the app insights bins on the Windows side.

Reply:
Probably, not.

Reply:
+1 on not making it fatal, right now looks like it is